decidim-consultations/db/migrate/20221118203655_add_weight_field_to_consultations.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddWeightFieldToConsultations < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_consultations, :weight, :integer, null: false, default: 0
```

Comment:
Same thing here, I would prefer not to add a new migration for this especially if we are not adding this functionality.